README.md

```diff
@@ -11,7 +11,7 @@ The types in the SARIF SDK are in the `Microsoft.CodeAnalysis.Sarif` namespace.
 
 The SARIF SDK provides a set of classes which represent the elements of the SARIF format. We refer to this as the "SARIF object model". The root type that represents a SARIF log file is `SarifLog`. Other types in the SARIF object model are `Result`, `PhysicalLocation`, _etc._.
 
-Note: The SARIF SDK's build process automatically generates the SARIF object model classes from the SARIF JSON schema, which you can find at [`src/Sarif/Schemata/sarif-schema.json`](https://github.com/microsoft/sarif-sdk/blob/main/src/Sarif/Schemata/sarif-2.1.0-rtm.5.json). Although these files do exist in the repo (under [`src/Sarif/Autogenerated`](https://github.com/Microsoft/sarif-sdk/tree/main/src/Sarif/Autogenerated)), you should never edit them by hand.
```

Comment:
>rtm.5 [](http://example.com/codeflow?start=262&length=5)

All `rtm.5` are replaced with `rtm.6`.
I will point out all other changes in the PR.